docs/getting_started.rst

```diff
@@ -429,11 +429,33 @@ i.e. a `~regions.CompoundPixelRegion` or `~regions.CompoundSkyRegion` object.
 * The ``^`` operator calls the ``__xor__`` method which calls the :meth:`~regions.Region.symmetric_difference` method
   to create a symmetric difference compound region.
 
+.. code-block:: python
 
-TODO:
-
-* A code example
-* Add image illustrating the compound regions
+    >>> from astropy.coordinates import Angle, SkyCoord
+    >>> from regions import CircleSkyRegion
+    >>> circle1 = CircleSkyRegion(
+    ...    center=SkyCoord(1,2, unit='deg', frame='galactic'),
+    ...    radius=Angle('5 deg')
+    ...    )
+    >>> circle2 = CircleSkyRegion(
+    ...    center=SkyCoord(-4,3, unit='deg', frame='galactic'),
+    ...    radius=Angle('3 deg'),
+    ...    )
+    >>> type(circle1 & circle2)
+    ... regions.core.compound.CompoundSkyRegion
```

Comment:
Usually one puts stuff as it would appear in the Python console.
I.e. only put `...` for input.
This line is output and should not have `...` or be indented.
Same for print statement below.